src/i_winmusic.c

```diff
@@ -265,8 +268,18 @@ static void StreamOut(void)
     MIDIHDR *hdr = &MidiStreamHdr;
     MMRESULT mmr;
 
+    memset(hdr, 0, sizeof(*hdr));
     hdr->lpData = (LPSTR)buffer.data;
     hdr->dwBytesRecorded = buffer.position;
+    hdr->dwBufferLength = buffer.size;
+
+    mmr = midiOutPrepareHeader((HMIDIOUT)hMidiStream, hdr, sizeof(MIDIHDR));
+    if (mmr != MMSYSERR_NOERROR)
+    {
+        MidiError("midiOutPrepareHeader", mmr);
```

Comment:
`MidiError` does not call `I_Error` so this code will continue on. This will result in eventually calling `midiOutUnprepareHeader` on a buffer that wasn't successfully prepared.

Reply:
With https://github.com/chocolate-doom/chocolate-doom/pull/1644/commits/580abf9ead2b773c5314c08c50250bdba562a3dd commit, we return from `StreamOut` and leave `buffer.prepared == false`, so the `midiOutUnprepareHeader` call will not occur.

In general, I don't think MIDI stream errors worth an `I_Error` call.